### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ALPANNA-GH**
- initiate portfolio project 
- setup linter
- prepared `readme.md` file
> some links work fine only after merge to master

- create the `index.html` structured for a mobile-first approach
- create a style sheet with the same approach
- 1st breakpoint 375px
- 2nd breakpoint 768px
- fix image position
- apply Autoprefixer for `CSS` file

### CODE REVIEW

**Review by skyv26 — COMMENTED** [review]
## Hi @ALPANNA-GH, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟨 




## Reason 🗒️ 
You have submitted a project, that doesn't have all the required header in the `README` file. Please update your `README` file with all headers because except the `FAQ` section or header rest everything is important to add because it will make your **README** looks more professional. There are six headers missing which are the part of Getting Started section.

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+          <li class="btn"><a href="#"> <img src="./images/Icon-Menu.svg" alt="menu-icon"></a></li>
+        </ul>
+      </nav>
+    </div>
+  </section>
 
-    <h1 class="headtitle">Hello Microverse</h1>
-    
+  <!-- body of the page  -->
+  <section class="main">
```
Comment (divinecharlotte) [inline index.html:28]: - [ ] Kindly consider giving this section the margins (top, bottom, left, right) like defined in the design to make your design an exact duplicate of the provided one check the images below for more clarification
- your design 👇 
Screen Shot 2023-06-06 at 18 58 33

- Figma design 👇 
Screen Shot 2023-06-06 at 19 00 30

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ALPANNA-GH       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍  
- Descriptive commit messages 👍 

 




 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -18,51 +30,111 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> see the hello microverse from here.
+> the link to the live portfolio.
 
-- [Live Demo Link](<a href="https://raw.githack.com/ALPANNA-GH/Hello-Microverse/HomePage/index.html" target="_blank" >Hello Microvers</a>)
+- [Live Demo Link](https://raw.githack.com/ALPANNA-GH/Portfolio/index.html)
```
Comment (usorfaitheloho) [inline README.md:81]: As suggested by the previous review, please remove this broken link or simply write `coming soon` from your README file so as not to have links that are not functional 👍

File: index.html [hunk 3]
```diff
@@ -1,15 +1,52 @@
...
+    <div class="social">
+      <ul class="social">
+        <li><a href="#"><img src="./images/github.svg" alt="github-icon" /></a></li>
+        <li><a href="#"><img src="./images/linkedin.svg" alt="linkedin-icon" /></a></li>
+        <li><a href="#"><img src="./images/angel.svg" alt="angel-icon" /></a></li>
+        <li><a href="#"><img src="./images/twitter.svg" alt="twitter.icon" /></a></li>
+        <li><a href="#"><img src="./images/medium.svg" alt="medium-icon" /></a></li>
+      </ul>
+    </div>
+    <!-- <div class="bodyImage"></div> -->
```
Comment (usorfaitheloho) [inline index.html:48]: You did well making the changes suggested by the previous reviewer, please remove this commented line of code so as to have a clean and easily maintainable code 😄

File: index.html (hunk 1 above)
Comment (usorfaitheloho) [reply index.html:28]: As suggested by the previous reviewer, kindly add the required margin-top for the `headline section` to enable you have a design very similar to the design given and a better UI

Figma design :
image

Your design:
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @ALPANNA-GH \ ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**